data/SugarBean.php

```diff
@@ -1919,7 +1919,15 @@ public function save($check_notify = false)
             foreach ($this->email_addresses_non_primary as $mail) {
                 $this->emailAddress->addAddress($mail);
             }
-            $this->emailAddress->save($this->id, $this->module_dir);
+            $this->emailAddress->saveEmail(
```

Comment:
Same here - Is this regression?

Reply:
$this->emailAddress is deprecated. Original issue is from this commit: https://github.com/salesagility/SuiteCRM/pull/5118/commits/0f01e1a17a3401dff4dadea261e661f85eac472e